scamp/scamp-isr.c

```
@@ -279,7 +290,7 @@ INT_HANDLER pkt_p2p_int(void)
     key &= 0xffff;
 
     if (p2p_type == P2P_LEVEL) {
-        p2p_region(data, key);
+        // Do Nothing - no longer supported!
```

Silent error eating is not really a good idea.